Geometry/CMSCommonData/data/materials/2015/v1/materials.xml

```diff
@@ -1204,23 +1204,23 @@
   <!--Phase1 BPIX micro-twisted pairs boundle material as defined by L. Caminada-->
   <CompositeMaterial name="micro_twisted_power_cable" density="3.639*g/cm3" symbol=" " method="mixture by weight">
     <MaterialFraction fraction="0.36933">
-      <rMaterial name="trackermaterial:T_Copper"/>
+      <rMaterial name="materials:Copper"/>
```

Comment:
@ianna @vargasa I understand that these changes are totally transparent, looking into https://cmssdt.cern.ch/lxr/source/Geometry/TrackerCommonData/data/trackermaterial.xml

Reply:
yes.